crates/oro-pack/src/lib.rs

```diff
@@ -1,3 +1,7 @@
+use async_std::fs::File;
+use async_std::io as AsyncIO;
```

Comment:
small style nitpick: Can you rename this to `async_io`?

Reply:
or `aio`, even tbh

Reply:
Done in d20ceeb !